src/wix/WixToolset.Core/Link/AddDefaultSymbolsCommand.cs

```diff
@@ -126,7 +127,10 @@ private void AddSymbols(IntermediateSection section, params IntermediateSymbol[]
                 {
                     var symbolWithSection = new SymbolWithSection(section, symbol);
                     var fullName = symbolWithSection.GetFullName();
-                    this.Find.SymbolsByName.Add(fullName, symbolWithSection);
+                    if (!this.Find.SymbolsByName.ContainsKey(fullName))
```

Comment:
Can't wait until we move to a C# version that supports the `.TryAdd()` method.

Reply:
Worse, it's `IDictionary`, which still doesn't have it...